### PR TITLE
Added sdl+vitagl example

### PR DESCRIPTION
**Pull request by nikarh**

### CODE REVIEW

**Comment by pheki** [comment]
Related references for some of the changes on this PR:
- https://github.com/rust-lang/rust/pull/126140
- https://github.com/rust-lang/rust/pull/115974

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Install Northfear SDL fork
+        run: |
+          chown -R 1000:1000 northfear-sdl2
+          cd northfear-sdl2
+          sudo --preserve-env=VITASDK -u \#1000 vita-makepkg
+          vdpm sdl2-2.30.2-1-arm.tar.xz
```
Comment (pheki) [inline .github/workflows/ci.yml]: What about something like:

```
PKGVER=$(grep pkgver= VITABUILD | cut -d '=' -f2)
PKGREL=$(grep pkgrel= VITABUILD | cut -d '=' -f2)
vdpm sdl2-$PKGVER-$PKGREL-arm.tar.xz
```

This way we don't need to remember to update the sdl2 version everywhere

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -17,6 +17,12 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Install Northfear SDL fork
+        run: |
+          chown -R 1000:1000 northfear-sdl2
+          cd northfear-sdl2
+          sudo --preserve-env=VITASDK -u \#1000 vita-makepkg
+          vdpm sdl2-2.30.2-1-arm.tar.xz
```
Comment (pheki) [inline .github/workflows/release.yml]: Same as above

File: .github/workflows/ci.yml (hunk 1 above)
Comment (nikarh) [reply .github/workflows/ci.yml]: I decided to go with the easiest solution and use glob, since there is only one such file in the folder :)

Comment (pheki) [reply .github/workflows/ci.yml]: Amazing!

**Review by pheki — APPROVED** [review]
:tada:

**Pull request merged**